### ISSUE TITLE
follow-up on reveal.js latest versions

### ISSUE THREAD

**Issue by parmentelat** [damianavila/RISE#560]
`reveal.js` has shipped a version 4.0

and the last version released in the 3.x family is 3.9.2 while we are running on top of 3.8.0

either way, an upgrade of our base `reveal.js` seems to be in order

**Comment by yangchino1** [comment]
RISE doesn't work in the most up-to-date conda environment, with python 3.7 or 3.8. The RISE button didn't show up on the notebook toolbar and all the keyboard shortcuts failed. Is this problem related to the update of `reveal.js`?

**Comment by damianavila** [comment]
I agree!!

**Comment by parmentelat** [comment]
as an update on this one, the latest reveal.js is now 4.1.0 released on oct 1 2020

the whole story is here <https://github.com/hakimel/reveal.js/releases>

**Comment by damianavila** [comment]
Yep, reveal 4 is most likely a pre-requisite for building the jupyterlab extension because of the multiple presentation support. More details in the relevant [PR](https://github.com/damianavila/RISE/pull/381#issuecomment-722029634) I plan to re-re-revive in the next few days...
